### PR TITLE
Fix a bug in the function name

### PR DESCRIPTION
**Pull request by shrit**
I am not sure if it is an error. Logically both of the functions should have the same name.

Signed-off-by: Omar Shrit <omar@shrit.me>

### CODE REVIEW

File: include/ensmallen_bits/callbacks/store_best_coordinates.hpp [hunk 1]
```diff
@@ -55,7 +55,7 @@ class StoreBestCoordinates
   //! Get the best coordinates.
   ModelMatType const& BestCoordinates() const { return bestCoordinates; }
   //! Modify the best coordinates.
-  ModelMatType& BestCoordinatesl() { return bestCoordinates; }
+  ModelMatType& BestCoordinates() { return bestCoordinates; }
```
Comment (zoq) [inline include/ensmallen_bits/callbacks/store_best_coordinates.hpp:58]: Nice catch!

**Review by zoq — APPROVED** [review]
Nice, thanks for the patch.

**Comment by rcurtin** [comment]
@zoq want to try the release script once you merge this? :smile:

**Review by birm — APPROVED** [review]
BestPatch() { return #230; }

**Comment by zoq** [comment]
> @zoq want to try the release script once you merge this? 😄

Yeah, good idea.

**Comment by zoq** [comment]
> > @zoq want to try the release script once you merge this? 😄
> 
> Yeah, good idea.

Let's include the optimizer report feature in the new release as well, will update the PR.

**Pull request merged**